pages/pydata.md

```diff
@@ -0,0 +1,30 @@
+title: PyData Track
+---
+
+This year, for the first time, the teams at PyCon and [PyData London](http://london.pydata.org/) are joining forces to deliver a data-science focussed track at the conference.
+***
+
+## What we are looking for
```

Comment:
It's not clear that this section is about the CFP.  You could add a line to the paragraph above saying something like: "We will be hosting a number of talks and workshops about data science at the the conference, and we are looking for members of the PyData community to submit proposals to the [conference CFP](https://hq.pyconuk.org/cfp/proposals/new/)."